install.dd

```diff
@@ -0,0 +1,204 @@
+Ddoc
+
+
+$(D_S $(TITLE),
+
+$(H2 Introduction)
+
+The $(LINK2 https://dlang.org/install.sh, `dlang.org/install.sh`) script is the
+official D version manager for DMD, GDC and LDC on FreeBSD, Linux and macOS.
+It provides a convenient way to fetch and install a D compiler
+without requiring administrative privileges nor external dependencies.
+Its features include:
+
+$(UL
+ $(LI Maintaining one or more portable compiler installations in the user's home directory (`~/dlang`))
+ $(LI Maintaining the latest stable version of the D package manager, $(LINK2 http://code.dlang.org/docs/commandline, `dub`))
+ $(LI Self-updating via `~/dlang/install.sh update`)
+ $(LI Supporting nested activation of compilers. Activating a compiler, will setup the `PATH`, `LIBRARY_PATH`, `LD_LIBRARY_PATH`, `DMD`, `DC`, and `PS1` environment variables, and a `deactivate` command, which reverts the modified variables to their old values.)
+ $(LI Supporting Fish shell, in addition to Bash-compatible shells.)
+ $(LI Verifying binary signatures via the $(LINK2 $(ROOT_DIR)gpg_keys.html,D keyring) (automatic if `gpg` is installed))
+ $(LI For `dmd`, installation of the following versions is supported:
+    $(UL
+        $(LI `dmd` - latest stable version)
+        $(LI `dmd-beta` - latest beta version)
+        $(LI `dmd-nightly` - the latest nightly release)
+        $(LI `dmd-<version>` - specific official release (including beta releases))
+        $(LI `dmd-branch` - experimental compiler branches)
+    )
+ )
+ $(LI For `ldc`, installation of the following versions is supported:
+    $(UL
+        $(LI `ldc` - latest stable version)
+        $(LI `ldc-beta` - latest beta version)
+        $(LI `ldc-<version>` - specific official release (including beta releases))
+    )
+ )
+ $(LI For `gdc`, installation of the following versions is supported:
+    $(UL
+        $(LI `gdc` - latest stable version)
+    )
+ )
+)
+
+$(H3 Content)
+
+$(UL
+    $(LI $(RELATIVE_LINK2 get, Downloading the installer))
+    $(LI $(RELATIVE_LINK2 usage, Usage))
+    $(UL
+        $(LI $(RELATIVE_LINK2 install, Installation))
+        $(LI $(RELATIVE_LINK2 uninstall, Uninstallation))
+        $(LI $(RELATIVE_LINK2 list, Listing available compilers))
+        $(LI $(RELATIVE_LINK2 update, Update))
+    )
+)
+
+$(H2 $(LNAME2 get, Downloading the installer))
+
+$(CONSOLE
+wget https://dlang.org/install.sh
```

Comment:
How about:

```shell
mkdir -p ~/dlang && wget https://dlang.org/install.sh -O ~/dlang/install.sh
```

Reply:
Yes, better to have a consistent install location, if the curl method installs it differently.